### DISCUSSION TITLE
Create a comprehensive list of Edge cases we need to cover.

### DISCUSSION THREAD

**Discussion by crazyrabbitLTC** [wesfloyd/pinception#10] [General]
- If the poster is malicious: How do we prove the supplied file is availible?

**Comment by wesfloyd** [comment]
Attack vectors discussed:
- Storage nodes sharing their answers to the challenge.
- Storage requesters are malicious - they withold the file from the IPFS p2p network.

**Comment by crazyrabbitLTC** [comment]
- how do we ensure no one can reuse other work? IE: they don't show they have the file but rather reuse someone elses work?
